recipes/bytewax-feedstock/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "bytewax" %}
+{% set version = "0.16.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/bytewax/bytewax/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 993fe70f3a206adc52aae1099c00e14a92ec55387dda4b0cda2b6f6fbf0ed35b
+  patches:
+    # this patch disable the vendored ssl/gssapi vendored libraries
+    # falling back to use the conda installed ones
+    - ssl.patch
+
+build:
+  skip: True  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - curl
+    - {{ compiler('cxx') }}
+    - make
+    - pkg-config
+    - python
+  host:
+    - cmake
+    - maturin
+    - rust
```

If you need the compiler you need to declare it like in https://github.com/conda-forge/cryptography-feedstock/blob/32ac6dabfba953884ee338e40e183fb2d1ed6ac2/recipe/meta.yaml#L67

```suggestion
    - rust
```